index.html

```diff
@@ -208,14 +212,18 @@ <h1>Skills</h1>
 
     <img class="illustration3" src="media\illustration-contact-form.svg" alt="illustration">
     
+    <div class="desktop-img">
+    <img class="desk-illustr1" src="./media/illustration-1-contact-form-desktop.svg" alt="">
+    <img class="desk-illustr2" src="./media/illustration-2-contact-form-desktop@2x.png" alt="">
+    <img class="desk-illustr3" src="./media/contact form-illustratoin-3-desktop.svg" alt="">
+    
```

Comment:
- [ ] As suggested by previous reviewers please align your skill frames with contact us card like that of in the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=s8WLg64fDsS6M8eZ-0)

#### **_Figma Design_**

<img width="733" alt="Screenshot 2023-02-24 at 10 09 19 AM" src="https://user-images.githubusercontent.com/44798044/221100204-f5ed624e-fffb-4607-8844-a134794533d8.png">

#### _**Your Design**_

<img width="1501" alt="Screenshot 2023-02-24 at 10 10 01 AM" src="https://user-images.githubusercontent.com/44798044/221100233-fbd4f5af-5019-4dff-acf6-594dfdf15ae3.png">